build.gradle

```diff
@@ -307,8 +307,8 @@ artifactory {
 
 dependencies {
     implementation "com.synopsys.integration:integration-rest:10.3.6"
-    implementation 'com.synopsys.integration:hub-imageinspector-lib:14.2.1'
```

Comment:
Not sure why git diff is showing existing version as 14.2.1 here. Master is at 14.3.2

Reply:
Yes, I too see that issue but in code I see 14.3.2.